library/foundations/scripts/templates/default-theme-classes.template.ts

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
new template to generate TS file with `defaultClassNames` and `imageBasePath` constants.